docs/user/generated_config.adoc

```diff
@@ -793,6 +793,13 @@ This config takes a map of crate names with the exported proc-macro names to ign
 --
 Internal config, path to proc-macro server executable.
 --
+[[rust-analyzer.progressReporting.verbosity]]rust-analyzer.progressReporting.verbosity (default: `[]`)::
++
+--
+Configures the level of detail rust-analyzer will report while scanning files.
+
+If not set, this will default to including parent directories while scanning.
```

Comment:
This feels a bit confusing (I'd expect `[]` to mean that nothing is reported, with `null` being the default).

But I also don't see what makes it behave like the docs say.

Reply:
Ah crud, I think I got the options here flipped. Lemme fix that.